modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
+You can add different extensions to your plug-in that are loaded at runtime.
+
+[id="adding-tab-to-pods-page"]
+== Adding a tab to the pods page
+You can add a tab to the *Pod Details* page.
+
+.Procedure
+
+. Edit the `console-extensions.json` file.
++
+[source,json]
+
+----
+}
```

Comment:
@spadgett For some reason, if you look at the [preview]( https://deploy-preview-41260--osdocs.netlify.app/openshift-enterprise/latest/web_console/dynamic-plug-ins.html#adding-tab-to-pods-page), it doesn't like the commas for #'s 1 and 2.

Reply:
> @spadgett For some reason, if you look at the [preview](https://deploy-preview-41260--osdocs.netlify.app/openshift-enterprise/latest/web_console/dynamic-plug-ins.html#adding-tab-to-pods-page), it doesn't like the commas for #'s 1 and 2.

Probably because they're JSON fragments that aren't valid JSON on their own.